core/src/test/java/com/linecorp/armeria/server/HttpResponseExceptionTest.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2017 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package com.linecorp.armeria.server;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.Test;
+
+import com.linecorp.armeria.common.HttpStatus;
+
+public class HttpResponseExceptionTest {
+    static {
+        System.setProperty("com.linecorp.armeria.verboseExceptions", "false");
```

Comment:
I think this will affect other tests. How about just removing this test or checking parameter validations only?